src/scenarios/personalization/ArrayAddFavorites/ArrayAddFavorites.tsx

```diff
@@ -0,0 +1,245 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import {
+  getMoviesQuery,
+  getTvShowsQuery,
+  setArrayItemMutation,
+} from './graphql-documents';
+
+interface entity {
+  id: string;
+  title: string;
+  type: string;
+}
+
+export const ArrayAddFavorites: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+
+  const [entityArray, setEntityArray] = useState<entity[]>([]);
+  const [entityId, setSelectedEntity] = useState<string>('');
+
+  const fetchAllCatalogItems = async (): Promise<void> => {
+    try {
+      const apolloClient = getApolloClient(
+        new URL('graphql', activeProfile.catalogServiceBaseURL).href,
+      );
+
+      const resultMovies = await apolloClient.query({
```

Comment:
Use a single query to get both movies and tv shows from the catalog service (refer my comment in the `[graphql-documents.ts` file